chains/evm/deploy.go

```diff
@@ -1,3 +1,6 @@
+// Copyright 2021 ChainSafe Systems
+// SPDX-License-Identifier: LGPL-3.0-only
+
 package evm
```

Comment:
No im am not talking about another repo.

Why this file previously was in package `core/e2e/evm` ? Because it was about preparing E2E testing. Why it was not in `chains/evm/...`. Because it was never used there. So why to put it int to the core folder of evm chain. Is it some abstraction that is used everywhere inside evm chain package? No i do not think so. It is still a set of functions to prepare E2E env. 
On the call we were discussing with you, that you need to add the CLI that is going to prepare (deploy) all necessary staff for local development (the same it does for E2E testing), so you need to use mostly the same code that e2e uses, and in order to prevent code duplication you need to move common low level function somewhere.

So function  PrepareEVME2EEnv right now uses both in E2E testings and in CLI that set ups local env, probably you should rename it? Think more about where this file could be placed. 
PS the best place to inspire is golang source code